doc/dataset.md

```diff
@@ -159,64 +153,70 @@ detailed definition is presented at
 
-The standardized crystal structure after {ref}`idealization <def_idealize_cell>` corresponding to a Hall symbol is stored in
-`n_std_atoms`, `std_lattice`, `std_types`, and
-`std_positions`. These output usually contains the rotation in Cartesian
+The standardized crystal structure after {ref}`idealization
+<def_idealize_cell>` corresponding to a Hall symbol is stored in
+``n_std_atoms``, ``std_lattice``, ``std_types``, and
+``std_positions``. These output usually contains the rotation in Cartesian
 coordinates and rearrangement of the order atoms with respect to the
 input unit cell.
 
 **At versions 1.7.x and 1.8 or before**, the variable names of the
-members corresponding to those above are `n_brv_atoms`,
-`brv_lattice`, `brv_types`, and `brv_positions`, respectively.
+members corresponding to those above are ``n_brv_atoms``,
+``brv_lattice``, ``brv_types``, and ``brv_positions``, respectively.
 
 (dataset_std_rotation_matrix)=
-
-### `std_rotation_matrix`
+### ``std_rotation_matrix``
 
 **New in version 1.11**
 
 Rotation matrix that rotates the standardized crystal structure
-before idealization {math}`( \mathbf{a}_\mathrm{s} \; \mathbf{b}_\mathrm{s} \; \mathbf{c}_\mathrm{s} )` to that after
-idealization {math}`( \bar{\mathbf{a}}_\mathrm{s} \; \bar{\mathbf{b}}_\mathrm{s} \; \bar{\mathbf{c}}_\mathrm{s} )` in
+before idealization $( \mathbf{a}_\mathrm{s} \;
+\mathbf{b}_\mathrm{s} \; \mathbf{c}_\mathrm{s} )$ to that after
+idealization $( \bar{\mathbf{a}}_\mathrm{s} \;
+\bar{\mathbf{b}}_\mathrm{s} \; \bar{\mathbf{c}}_\mathrm{s} )$ in
 Cartesian coordinates of the given input unit cell. The rotation
-matrix {math}`\boldsymbol{R}` is defined by
-
-```{math}
-:label: rotation-in-standardization
+matrix $\boldsymbol{R}$ is defined by
+$$
 
 ( \bar{\mathbf{a}}_\mathrm{s} \;
 \bar{\mathbf{b}}_\mathrm{s} \; \bar{\mathbf{c}}_\mathrm{s} )
 = ( \boldsymbol{R} \mathbf{a}_\mathrm{s} \;
 \boldsymbol{R} \mathbf{b}_\mathrm{s} \; \boldsymbol{R}
 \mathbf{c}_\mathrm{s} ).
-```
-
-More precisely, this rotation matrix is an orthonormal matrix. Since {math}`( \mathbf{a}_\mathrm{s} \; \mathbf{b}_\mathrm{s} \; \mathbf{c}_\mathrm{s} )` can
-be distored, to make {math}`\boldsymbol{R}` orthonormal, it is calculated as
-follows. Make cubes of {math}`( \mathbf{a}_\mathrm{s} \; \mathbf{b}_\mathrm{s} \; \mathbf{c}_\mathrm{s} )` and {math}`( \bar{\mathbf{a}}_\mathrm{s} \; \bar{\mathbf{b}}_\mathrm{s} \; \bar{\mathbf{c}}_\mathrm{s} )` by
-{math}`\mathrm{L} = \begin{pmatrix}\frac{\mathbf{a}}{|\mathbf{a}|} & \frac{(\mathbf{a} \times \mathbf{b}) \times \mathbf{a}}{|(\mathbf{a} \times \mathbf{b}) \times \mathbf{a}|} & \frac{\mathbf{a} \times \mathbf{b}}{|\mathbf{a} \times \mathbf{b}|} \end{pmatrix}`. Watching
-{math}`\mathrm{L}`s as {math}`3\times 3` matrices, {math}`\boldsymbol{R}` is
-obtained by solving {math}`\bar{\mathrm{L}}_\mathrm{s} = \boldsymbol{R} \mathrm{L}_\mathrm{s}`.
-
-### `std_mapping_to_primitive`
+$$ (rotation-in-standardization)
```

Comment:
This equation does not seem to be rendered correctly. Maybe a blank line at 197 is the culprit?

Reply:
It should be the label name because they should be converted to underscore